pkg/binding/backend/interactive.go

```diff
@@ -98,6 +102,38 @@ loop:
 	return selectedName, selectedGVK, nil
 }
 
+// SelectNamespaceForServiceInstances returns the namespace which services instances should be listed from.
+// An empty return value means that service instances will be listed from the current namespace.
+// Otherwise, they will be listed from the specified namespace.
```

Comment:
The comment seems copy-pasted from `interface.go` file. IMO, having it at one place is OK. In an earlier discussion (on some PR review) with Philippe, he suggested putting it in `interface.go` file and only adding implementation specific comment at the places where it is implemented. It made sense to me.

Reply:
Makes sense. I updated the comments in the interface, but also in each different implementation.